### DISCUSSION TITLE
Cuando es la entrega de la practica de la pasteleria

### DISCUSSION THREAD

**Discussion by JonathanGrimaldo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#229] [Q&A] [answered]
Y otra duda, la entrega sera en un formulario como el de forms?

**Comment by doguedogue** [comment]
Mañana 4 de marzo

Si van a mandar un forms asi como todas las practicas / katas

**Comment by JuanSalvador73** [comment, marked as answer]
La entrega es el viernes, pero se ha hablado de que se cambie al domingo 6.

**Reply by JorgePadronO** [reply]
Ojala, he estado ocupado con trabajos de la escuela..... apenas haré mi ejercicio
